src/events.ts

```diff
@@ -455,3 +455,20 @@ export interface AdManifestLoadedEvent extends Event {
    */
   downloadTime: number;
 }
+
+export interface VideoPlaybackQualityChangedEvent extends Event {
+  newVideoQuality: {
+    height?: number;
+    width?: number;
+    frameRate?: number;
+    bitrate?: number;
+    codec?: string;
+  };
```

Comment:
I think we can define a type (`VideoQuality`) for this like it is done for the `SubtitleTrack`.
Then we do not have to define the structure here twice :)

Also please add some documentation as it is done with the other events and their properties. (The API reference of the underlying SDKs is probably a good resource for documentation: [Android](https://cdn.bitmovin.com/player/android/3/docs/index.html), [iOS](https://cdn.bitmovin.com/player/ios/3/docs/index.html))

Reply:
@matamegger do you have a preference on structure?  If it were up to me I'd place this type in a `media.ts` file so that there is a definitive place to put other things like it, `AudioQuality` for example.  This is effectively the same package structure of the Java library, where those types are under the `com.bitmovin.player.api.media.*` pacakges.

Reply:
@matamegger I put it under media, as well as fixed the fact that android wasn't converting the event properties for the VideoPlaybackQualityChangedEvent.  Let me know if you want the type somewhere else.

Reply:
Packing them into a common file seems fine 👍 

(cc @pogist )